Lib/fontbakery/profiles/fontwerk.py

```diff
@@ -31,22 +36,39 @@ def leave_this_one_out(checkid):
         'com.google.fonts/check/vendor_id',
         'com.google.fonts/check/fstype',
         'com.google.fonts/check/gasp',
+        'com.google.fonts/check/font_copyright',
+        'com.google.fonts/check/name/line_breaks',  # can be ignored because of general fontwerk name table test.
+        'com.google.fonts/check/varfont/has_HVAR',  # not needed for fontwerk fonts.
+        'com.google.fonts/check/glyf_nested_components',  # it seem to be not an issue anymore. Maybe downgrade to 'WARN'. Tested it in various browsers, InDesign, exported a PDF, printed it.
```

Comment:
They don't print well with postscript printers (still quite in used by professionals).

Reply:
@RosaWagner Thank you very much. I was not aware of that. In that case, would it make sense to handle Desktop fonts and Webfonts differently? We save a lot of kb in variable fonts, when we keep the composites and as it's 'just' an issue for prints, it would be ok to keep the composites in web fonts and decompose in desktop fonts.

Reply:
the cool thing that fontmake/builder does is that it "flattens" the nested components, so they don't get decomposed, they get reassigned to the original component during export. Same option in GlyphsApp would be great (https://forum.glyphsapp.com/t/parameter-to-flatten-nested-components/20888) btw. Since you can also export and print pdfs from a web page, I don't know how far we can go to define different spec according to the environment.